content_handler/vulkan_surface.h

```diff
@@ -43,18 +44,17 @@ class VulkanSurface : public flow::SceneUpdateContext::SurfaceProducerSurface,
       std::function<void(void)> on_writes_committed) override;
 
   // |flow::SceneUpdateContext::SurfaceProducerSurface|
-  uint32_t GetSessionImageID() const override;
+  mozart::client::Image* image() override;
```

Comment:
Why is this virtual anyhow?  But since it is, per Google C++ style guide, this should be called GetImage().  Sorry for the churn.